lib/chatkit/client.rb

```diff
@@ -769,11 +782,44 @@ def update_permissions_for_role(options)
       })
     end
 
+    def upload_attachment(token, room_id, part)
+      body = part[:file]
+      content_length = body.length
+
+      if content_length <= 0
+        raise Chatkit::MissingParameterError.new("File contents size must be greater than 0")
+      end
+
+      attachment_req = { content_type: part[:type],
+                         content_length: content_length
+                       }
```

Comment:
This is stylistic only, and on that front the codebase is not in good shape, but I would definitely present

```
      attachment_req = { 
        content_type: part[:type],
        content_length: content_length
      }
```

Reply:
Yes. My editor wants to indent differently. My options are
```ruby
attachment_req = { content_type: part[:type],
                   content_length: content_length
                 }
```
or
```ruby
attachment_req =
  { content_type: part[:type],
    content_length: content_length
  }
```

Perhaps there's a tool that can enforce a consistent style, I'll have a look.

Reply:
Hah, you need to have a serious word with your editor, both of those are awful ;)

Reply:
ehm, actually the editor agrees with you :grimacing: